Ниндзя: Цель на сбор вампирской крови, мидраунд moderate ивент на ниндзя и другие правки.

Furukia opened the pull request:
<!-- Пишите **НИЖЕ** заголовков и **ВЫШЕ** комментариев, иначе ваш текст может не отобразиться. -->
<!-- В Contributing.MD вы можете найти некоторые рекомендации к оформлению пулл-реквеста. -->

## Описание
<!-- Опишите, что делает ваш ПР. Документировать каждую деталь не требуется, просто укажите основные изменения. -->
### Кратко
Новая цель + теперь ниндзя раундстартовый не спавнится один и ВСЕГДА сопровождается минорами. 
Так же теперь добавлен мидраунд ивент на ниндзя для гостов. Что даст большему количеству людей опробовать антагониста.

### Чейнджлог
* Создана Blood Sample Machine (BSM) предназначенная для выполнения ниндзя цели на сканирование крови вампиров. 
* Создана вышеназванная цель
* Добавлен tts_seed машинам и костюму ниндзя
* BSM, виалы и шприцы добавлены на карту в пещеру к ниндзя.
* TGUI интерфейс BSM
* Убраны найденные старые комменты с логами.
* Был переработан порядок выдачи целей и они были разделены на 4 блока - "generic", "protector", "hacker", "killer" вместо прошлого "stealthy", "generic", "aggressive" 
generic - теперь единственный блок который генерирует цели для ниндзя без других антагов. Используется при спавнах админами или мидраунд.
* Добавлен Мидраунд ивент на ниндзя
* Добавлена частичная защита от некоторых способностей вампира для ниндзя. Её можно получить выполнив цель.
* В список целей для ниндзя так же добавлена цель на избиение цели (pain_hunter от пупса) для разнообразия
* В One-Click Antag до предложения гостам - теперь предлагается админу написать кастомную цель ниндзя, если он того пожелает.
* Добавлено несколько дополнительных подсказок к целям и не только в интерфейс костюма ниндзя.
* ~~Сместила подсказки ниже в интерфейсе костюма.~~

## Кратко о видах целей
### protector
- Защита(трейтухи)
- Киборг(50%)
- Подстава
- 2 кражи
- Боль(pain hunter)
- Выжить

### hacker
- Кровь(Вампиры)
- Киборг(50%)
- Взлом ИИ/Взлом Серверов
- Похищения/Деньги
- 1 кража/1 убийство
- Выжить

### killer
- Паразиты(Генки)
- Киборг(50%)
- Бомба
- Похищения
- 2 убийства
- Выжить

### generic
- Киборг(50%)
- Взлом ИИ/Взлом Серверов
- Бомба(25%)/Подстава(25%)/Боль(50%)
- Похищения/Деньги
- 2 кражи/ 2 убийства
- Выжить
(Возможность зафорсить 1 кастом цель при спавне админами вместо всех целей этого типа выше)

## Демонстрация изменений
<!-- Здесь вы можете показать изменения внешне, к примеру новые спрайты, звуки или изменения карты. Или написать, что именно изменилось для игроков. Этот пункт полностью опционален и его можно удалить. -->
Интерфейс BSM:
![2023-02-20_22-05-14](https://user-images.githubusercontent.com/67807868/220470694-cc4824f6-501e-4814-800f-1120735f8508.png)
Сама машина в пещере:
![ninja_cave](https://user-images.githubusercontent.com/67807868/220471429-d6b3a2f3-64b3-4f41-a8cd-95574d6d81d9.png)

Furukia committed:
Цель на кровь + BSM + мелкие правки

* Создана Blood Sample Machine (BSM) предназначенная для выполнения ниндзя цели на сканирование крови вампиров. (Дорабатывается)
* Создана вышеназванная цель
* Цель добавлена в генерацию generic целей
* Добавлен tts_seed машинам и костюму ниндзя
* BSM, виалы и шприцы добавлены на карту в пещеру к ниндзя.
* TGUI интерфейс устройства (Дорабатывается)
* Убраны найденные старые комменты с логами.

Furukia committed:
BSM, moderate ивент и переработка выдачи целей

* Был переработан порядок выдачи целей и они были разделены на 4 блока - "generic", "protector", "hacker", "killer" вместо прошлого "stealthy", "generic", "aggressive"
generic - единственный блок который генерирует цели для ниндзя без других антагов. Используется при спавнах админами или мидраунд.
* Добавлен мидраунд ивент на ниндзя
* Доделана цель на добычу вампирской крови. (Как же я устала её тестить  эти 4 дня, ля)
* Добавлена частичная защита от некоторых способностей вампира для ниндзя. Её можно получить выполнив цель.
* В список целей для ниндзя так же добавлена цель на избиение цели(pain hunter от пупса) для разнообразия
* В One-Click Antag до предложения гостам - предлагается админу написать кастомную цель ниндзя, если он того пожелает.
* Добавлено несколько дополнительных подсказок к целям и не только в интерфейс костюма ниндзя.
* Сместила подсказки ниже в интерфейсе костюма.

Den109G commented:
Некоторые "подсказки" уходят вникуда.
![image](https://user-images.githubusercontent.com/87372121/220569402-8a43b0dd-ec41-47ef-93cc-6aa969cc9442.png)

Furukia commented:
> Некоторые "подсказки" уходят вникуда. ![image](https://user-images.githubusercontent.com/87372121/220569402-8a43b0dd-ec41-47ef-93cc-6aa969cc9442.png)

Да ля... Я же их проверяла... пойду снова чекну

Furukia committed:
Правка подсказок

Ох уж эти тултипы и их размеры...
В конечном итоге я снова поместила подсказки наверх, потому что там они динамически не сдвигаются. И уменьшила размер кнопок чтобы все тултипы умещались в экран.
